ts/src/pro/coinbase.ts

```diff
@@ -73,7 +73,7 @@ export default class coinbase extends coinbaseRest {
             messageHash = messageHash + '::' + symbol.join (',');
         } else if (symbol !== undefined) {
             market = this.market (symbol);
-            messageHash = name + '::' + market['id'];
+            messageHash = name + '::' + symbol;
```

Comment:
in different areas, `messageHash` was constructed with marketId's , but in other places, with `symbol`. I've uniformed so everywhere `symbol` is used for `messageHash`